code/src/layouts/header/components/header-search-input-dropdown/HeaderSearchInputDropdown.tsx

```diff
@@ -68,19 +82,20 @@ const HeaderSearchInputDropdown = ({
 
   if (isError) {
     content = errorLabel;
-  } else if (isLoading) {
+  } else if (isLoading || totalElements === undefined) {
     content = loadingLabel;
```

Comment:
in this case if `totalElements === undefined` true will be `loadingLabel` ? How it works ?

Reply:
When we receive products with totalElements: 0 it means, that the response by our search param has been without results. But we have the stage, when we are typing a search query, and we need some fallback. Actually when we are typing the search query totalElements variable is `undefined`. That's why I did it